### ISSUE TITLE
Integration into "My Page"

### ISSUE THREAD

**Issue by fs5** [dalyons/redmine-todos-scrum-plugin#11]
It would be great if I could add my todo list to the "my page"

**Comment by dalyons** [comment]
I agree! Ill see if theres a hook for it in the plugin API, if so ill try and implement this when i get a chance!

**Comment by meglio** [comment]
Yes, it's good idea. Even better if we can check/uncheck an option to show my todo list on the right hand panel in redmine, read more in my feature proposition at redmine.org:

http://www.redmine.org/issues/5965
